spec/fixtures/files/invalid_subjects.csv

```diff
@@ -0,0 +1,2 @@
+IMPORT_ID,TYPE,TEXT,CORRECT_ANSWERS,ANSWER_1,ANSWER_2,ANSWER_3,ANSWER_4,ANSWER_5,ANSWER_6,ANSWER_7,KEYWORD,SUBJECT,LEVEL
```

Comment:
This csv goes through the importer_csv_spec to verify that it will not persist the records with invalid subjects.